packages/core/src/http-middlewares/before/prepare-request.js

```diff
@@ -148,33 +148,44 @@ const prepareRequest = function (req) {
 
   req = sugarBody(req);
 
-  // apply app requestTemplate to request
-  if (req.merge) {
-    const requestTemplate = (input._zapier?.app || {}).requestTemplate;
-    req = requestMerge(requestTemplate, req);
-  }
-
-  const replaceable = {
-    url: req.url,
-    headers: req.headers,
-    params: req.params,
-    body: req.body,
-  };
-
-  if (req[REPLACE_CURLIES]) {
-    // replace {{curlies}} in the request
+  if (req[REPLACE_CURLIES] || req.merge) {
     const bank = createBundleBank(
-      input._zapier.app,
-      input._zapier.event,
+      input?._zapier?.event || {},
       req.serializeValueForCurlies,
     );
-    req = {
-      ...req,
-      ...recurseReplaceBank(replaceable, bank),
+
+    const replaceable = {
```

Comment:
```suggestion
    let replaceable = {
```

Reply:
Good callout! Instead of reusing the same variable name, I've changed to use two different variable names in [b8ae466](https://github.com/zapier/zapier-platform/pull/1034/commits/b8ae466fc60a736d485a4b03e62d2718c102464b).